data/json/items/armor.json

```diff
@@ -11773,6 +11823,31 @@
     },
     {
         "type" : "ARMOR",
+        "id" : "shoes_birchbark",
+        "name" : "pair of birchbark shoes",
+        "name_plural" : "pairs of birchbark shoes",
+        "weight" : 345,
+        "color" : "light_grey",
+        "covers" : ["FEET"],
+        "to_hit" : 0,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "Simple shoes woven from strips of birch bark.  Not very confortable, but fairly water-resistant.",
+        "price" : 3500,
+        "material" : ["wood", "null"],
+        "volume" : 2,
+        "cutting" : 0,
+        "warmth" : 10,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 15,
+        "bashing" : -3,
+        "flags" : ["VARSIZE", "WATERPROOF"],
```

Comment:
"WATERPROOF" means "you don't get that body part wet while wearing it". Pretty sure your feet will get wet in shoes made from bark.